addons/website/views/website_templates.xml

```diff
@@ -299,7 +317,7 @@
 </template>
 
 <template id="navbar_nav" name="Navbar Nav">
-    <ul t-att-id="not is_mobile and 'top_menu'" t-attf-class="nav navbar-nav top_menu #{'' if _no_autohide_menu_mobile else 'o_menu_loading'} #{_nav_class}" role="menu">
+    <ul t-att-id="not is_mobile and 'top_menu'" t-attf-class="nav navbar-nav top_menu #{'' if _no_autohide_menu_mobile else 'o_menu_loading'} #{is_in_offcanvas and 'o_mega_menu_is_offcanvas mx-n3'} #{_nav_class} #{(_navbar_pills and not _navbar_no_pills) and 'nav-pills'}" role="menu">
```

Comment:
I don't know a condition such as `_navbar_pills and not _navbar_no_pills` would be needed :scream: